web/impact/requirements/mc.txt

```diff
@@ -0,0 +1 @@
+git+https://github.com/masschallenge/django-accelerator.git@AC-4598#egg=django-accelerator
```

Comment:
We're going to trip over the fact that we need to set the @branch in two different places. Maybe there's a way to set it once?

Reply:
This is a deploy issue that is supposed to be resolved by Shankar's ticket.  Won't fix.